web/package.json

```diff
@@ -32,12 +37,11 @@
         "webpack-cli": "^4.9.1"
     },
     "scripts": {
-        "bootstrap": "npm install && lerna bootstrap --hoist",
-        "build": "lerna run build --stream",
+        "build": "npm run build --workspaces",
         "build:debug": "cross-env NODE_ENV=development CARGO_FEATURES=avm_debug npm run build",
```

Comment:
Shouldn't the `--workspaces` argument also have been added on this line?

Reply:
It's not needed because it uses `build` that does `npm run build --workspaces`.

Reply:
I see, thanks. It's still not entirely clear to me "which" `build` script where runs when and why. I might read up on it. Until then, I'll remove it from #5834.